evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -822,3 +822,20 @@ class LinearPipelineDropDates(pipeline_class):
         assert set(explanation['drill_down']['currency']['feature_names']) == EXPECTED_CURRENCY_FEATURES
         assert set(explanation['drill_down']['provider']['feature_names']) == EXPECTED_PROVIDER_FEATURES_OHE
         assert set(explanation['drill_down']['datetime']['feature_names']) == {"datetime_year", "datetime_day_of_week"}
+
+
+@pytest.mark.parametrize("is_binary", [True, False])
```

Comment:
Do you think there is any value in adding multiclass or time series here?

Reply:
Added just to be safe!

Reply:
Can't do coverage for time series, since SKLearn stacked ensembler can't build upon our time series class, but added cov for multiclass